pkg/build/strategies/sti/sti.go

```diff
@@ -234,33 +234,33 @@ func (b *STI) PostExecute(containerID, location string) error {
 		Command:     append([]string{}, runCmd),
 		Env:         buildEnv,
 		ContainerID: containerID,
-		Repository:  b.request.Tag,
+		Repository:  b.config.Tag,
 	}
 
 	imageID, err := b.docker.CommitContainer(opts)
 	if err != nil {
-		return errors.NewBuildError(b.request.Tag, err)
+		return errors.NewBuildError(b.config.Tag, err)
 	}
 
 	b.result.Success = true
 	b.result.ImageID = imageID
 
-	if len(b.request.Tag) > 0 {
-		glog.V(1).Infof("Successfully built %s", b.request.Tag)
-		glog.V(1).Infof("Tagged %s as %s", imageID, b.request.Tag)
+	if len(b.config.Tag) > 0 {
+		glog.V(1).Infof("Successfully built %s", b.config.Tag)
+		glog.V(1).Infof("Tagged %s as %s", imageID, b.config.Tag)
```

Comment:
Same here, collapse these two. You can use "\n" if you really need two lines but I think we can just say:

"Successfully build %s and tagged as %s"